sample/db/samples/payment_methods.rb

```diff
@@ -2,7 +2,6 @@
   {
     :name => "Credit Card",
     :description => "Bogus payment gateway for development.",
-    :environment => "development",
```

Comment:
This is probably the issue you found @jordan-brough. Creating 4 credit card payment methods. We probably want to change the description also.